lib/services/discourse-types.d.ts

```diff
@@ -0,0 +1,16 @@
+import {
+    ServiceEmitContext,
+} from './service-types';
+
+export interface DiscourseMessageEmitContext extends ServiceEmitContext {
+    raw: string,
```

Comment:
Can you run TSLint over the branch, please (`gulp tslint`). It currently shows up several issues, including that semi-colons aren't being used for interface key delimiting:
```
raw: string;
```
We have a TS guide which should be followed: https://github.com/resin-io/process/blob/master/process/TypeScript_Coding_Guide.md

I'm fully aware that the entire ProcBots code needs going over to now ensure that it conforms to the guide (for example, tabs vs spaces), but any new code going in I'm trying to write to process spec.